src/Moryx/Workflows/API/Instance/ITokenHolder.cs

```diff
@@ -12,7 +12,7 @@ namespace Moryx.Workplans
     public interface ITokenHolder
     {
         /// <summary>
-        /// Workflow unique id of this holder. Same id can be present in another workflow
+        /// Workplan-unique id of this holder. Same id can be present in another workplan
```

Comment:
this is probably for a workplan instance

Reply:
Actually worlplan ist fine here, as all instances of the same workplan have the same Ids for the steps, it is unique within the plan, not within all plans.